sklearn/preprocessing/data.py

```diff
@@ -1698,14 +1698,11 @@ class OneHotEncoder(BaseEstimator, TransformerMixin):
 
     Parameters
     ----------
-    n_values : 'auto', int or array of ints
-        Number of values per feature.
+    classes : 'auto', 2D array of ints or strings or both.
```

Comment:
Look: while supported, numpy isn't really at home with strings. While I agree this seems to be a feature people want, we still want to support development (and where avoidable, not break already working code) for people using numpy more idiomatically!